extensions/tmtv/src/getHangingProtocolModule.js

```diff
@@ -173,425 +186,110 @@ const ptCT = {
-                target: true,
-              },
-              {
-                type: 'voi',
-                id: 'ctWLSync',
-                source: true,
-                target: true,
-              },
-            ],
-          },
-          displaySets: [
-            {
-              id: 'ctDisplaySet',
-            },
-          ],
-        },
-        {
-          viewportOptions: {
-            viewportId: 'ptAXIAL',
-            viewportType: 'volume',
-            background: [1, 1, 1],
-            orientation: 'axial',
-            toolGroupId: 'ptToolGroup',
-            initialImageOptions: {
-              // index: 5,
-              preset: 'first', // 'first', 'last', 'middle'
-            },
-            syncGroups: [
-              {
-                type: 'cameraPosition',
-                id: 'axialSync',
-                source: true,
-                target: true,
-              },
-              {
-                type: 'voi',
-                id: 'ptWLSync',
-                source: true,
-                target: true,
-              },
-              {
-                type: 'voi',
-                id: 'ptFusionWLSync',
-                source: true,
-                target: false,
-              },
-            ],
-          },
-          displaySets: [
-            {
-              options: {
-                voi: {
-                  windowWidth: 5,
-                  windowCenter: 2.5,
-                },
-                voiInverted: true,
-              },
-              id: 'ptDisplaySet',
-            },
-          ],
-        },
-        {
-          viewportOptions: {
-            viewportId: 'ptSAGITTAL',
-            viewportType: 'volume',
-            orientation: 'sagittal',
-            background: [1, 1, 1],
-            toolGroupId: 'ptToolGroup',
-            syncGroups: [
-              {
-                type: 'cameraPosition',
-                id: 'sagittalSync',
-                source: true,
-                target: true,
-              },
-              {
-                type: 'voi',
-                id: 'ptWLSync',
-                source: true,
-                target: true,
-              },
-              {
-                type: 'voi',
-                id: 'ptFusionWLSync',
-                source: true,
-                target: false,
-              },
-            ],
-          },
-          displaySets: [
-            {
-              options: {
-                voi: {
-                  windowWidth: 5,
-                  windowCenter: 2.5,
-                },
-                voiInverted: true,
-              },
-              id: 'ptDisplaySet',
-            },
-          ],
+        ctAXIAL,
+        ctSAGITTAL,
+        ctCORONAL,
+        ptAXIAL,
+        ptSAGITTAL,
+        ptCORONAL,
+        fusionAXIAL,
+        fusionSAGITTAL,
+        fusionCORONAL,
+        mipSAGITTAL,
+      ],
+      createdDate: '2021-02-23T18:32:42.850Z',
+    },
+    {
```

Comment:
Can we add the convention that stages get comments on them, or maybe even that they get their own const and just get included?  I find it hard to notice when going between stages, so I'd prefer to have a bit more separation.  Not something I've done before, but something I think we should do.